MAINTAINERS

```diff
@@ -11,4 +11,4 @@ Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
 Shiwei Zhang <shizh@microsoft.com> (@shizhMSFT)
 Toddy Mladenov <toddysm@gmail.com> (@toddysm)
 Vani Rao <vaninrao@amazon.com> (@vaninrao10)
-Yi Zha <yizha1@microsoft.com> @yizha1
+Yi Zha <yizha1@microsoft.com> @yizha1
```

Comment:
Please add an empty new line at the end per convention

Reply:
Updated.